run-integration-tests

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -o errexit
+trap 'echo "Aborting due to errexit on line $LINENO. Exit code: $?" >&2' ERR
+set -o errtrace
+set -o pipefail
+
+_report_destination=${ARTIFACTS:-/dev/null}
+
+micropipenv install --dev
```

Comment:
Is this run in the delivered container image from aicoe-ci? In that case, dependencies should be already present in the container image.